### PR TITLE
Upgrade to stretch

### PR DESCRIPTION
**Pull request by firecyberice**
This is waiting for: [image-builder PR2](https://github.com/hypriot/image-builder/pull/2)

### CODE REVIEW

**Comment by kraeml** [comment]
Works for me with https://github.com/DieterReuter/image-builder-rpi64. Only the python package was missed. May you add it

```bash
@@ -24,7 +24,7 @@ HYPRIOT_OS_VERSION="${HYPRIOT_OS_VERSION:-dirty}"
 ROOTFS_DIR="/debian-${BUILD_ARCH}"
 DEBOOTSTRAP_URL="http://ftp.debian.org/debian"
 DEBOOTSTRAP_KEYRING_OPTION=""
-DEFAULT_PACKAGES_INCLUDE="apt-transport-https,avahi-daemon,bash-completion,binutils,ca-certificates,curl,git-core,htop,locales,net-tools,ntp,openssh-server,parted,sudo,usbutils,wget,libpam-systemd"
+DEFAULT_PACKAGES_INCLUDE="apt-transport-https,avahi-daemon,bash-completion,binutils,ca-certificates,curl,git-core,htop,locales,net-tools,ntp,openssh-server,parted,sudo,usbutils,wget,libpam-systemd,python"
 DEFAULT_PACKAGES_EXCLUDE="debfoster"
 
 if [[ "${VARIANT}" = "raspbian" ]]; then
(END)
```

**Comment by firecyberice** [comment]
ping @DieterReuter

**Comment by firecyberice** [comment]
@kraeml I think the base rootfs should stay as small as possible. The python package can be added in the rpi-image-builder64 repository.

**Comment by DieterReuter** [comment]
I would also prefer to keep additional packages, like python, out of the base os-rootfs. These packages can be easily added later on!

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,10 @@ RUN apt-get update && apt-get install -y \
     --no-install-recommends && \
     rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update && apt-get install -y \
+gpg
```
Comment (DieterReuter) [inline Dockerfile]: Please install gpg with the install step before, saving time and layer size!

File: builder/files/usr/local/bin/hypriot-firstboot [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# shellcheck disable=SC1090
```
Comment (DieterReuter) [inline builder/files/usr/local/bin/hypriot-firstboot]: I like that.

File: Dockerfile (hunk 1 above)
Comment (firecyberice) [reply Dockerfile]: ups sorry it was for testing locally :-)

**Comment by kraeml** [comment]
@firecyberice I will send this in rpi-image-builder64. 

Thumps up for merging.

**Comment by firecyberice** [comment]
Can we merge this? @DieterReuter

**Pull request merged**